src/Grids/Grids.jl

```diff
@@ -92,5 +106,6 @@ include("grid_utils.jl")
 include("input_validation.jl")
 include("regular_cartesian_grid.jl")
```

Comment:
Maybe not now but in the future will we change the name of `regular_cartesian_grid.jl`?